samples/21_sound_and_effects.cr

```diff
@@ -22,35 +26,38 @@ activekey = [] of LibSDL::Keycode
 loop do
   case event = SDL::Event.wait
   when SDL::Event::Quit
-    #SDL::Mixer.unload_music(music)
+    music.stop
+    SDL::MIX.close
```

Comment:
Well, actually, they do different things. `quit` closes out the decoders, and `close` closes out the music player. However, the original sample I took this from only calls `Mix_Quit()`. I'm assuming you would use the `MIX.close` when you want to change songs? I'll remove that line since it's not necessary for this sample.